src/pages/people.js

```diff
@@ -35,11 +51,14 @@ export class PagePeople extends LitElement {
       <h2>People</h2>
       ${this._loading ? html`<p>Loading...</p>` : null}
       ${this._error ? html`<p>${this._error}</p>` : null}
-      <ul>
-        ${this._people.map((person) => html`<li>${person.name}</li>`)}
-      </ul>
+      <div>
+        ${this._people.map(
+          (person) => html`<person-item .val=${person}></person-item>`
+        )}
+      </div>
```

Comment:
Creo que sería mejor dejar el `ul` y los `li` para preservar la semántica del HTML.

```suggestion
      <ul>
        ${this._people.map(
          (person) => html`<li><person-item .val=${person}></person-item></li>`
        )}
      </ul>
```